internal/pkg/services/graphql-service.go

```diff
@@ -0,0 +1,42 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package services
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+const serviceName = "cartesi-rollups-graphql-server"
+
+type GraphQLService struct{}
+
+func (g *GraphQLService) Start(ctx context.Context) error {
```

Comment:
```suggestion
func (g GraphQLService) Start(ctx context.Context) error {
```

This doesn't need to be a pointer.